yt_dlp/extractor/cbc.py

```diff
@@ -94,9 +95,10 @@ class CBCIE(InfoExtractor):
         'skip': 'Geo-restricted to Canada',
     }, {
         # multiple CBC.APP.Caffeine.initInstance(...)
+        # currently doesn't work, needs a fix.
         'url': 'http://www.cbc.ca/news/canada/calgary/dog-indoor-exercise-winter-1.3928238',
         'info_dict': {
-            'title': 'Keep Rover active during the deep freeze with doggie pushups and other fun indoor tasks',
+            'title': 'Keep Rover active during the deep freeze with doggie pushups and other fun indoor tasks | CBC News',
```

Comment:
The title we want is the old one. It's the extractor that's wrong, not the test.

Reply:
OK, will revert.